naughts_crosses.py

```diff
@@ -144,13 +144,52 @@ def hasWinnerHorizontally():
 
     return False
 
+def hasWinnerDiagonally():
+    global board
+    global playerCounters
+    global ROWS
+    global COLLUMNS
+
+    for col in range(1, COLLUMNS + 1):
+        # | If the column is empty then it can just be skipped.
+        rowsHigh = len(board[col])
+        if not rowsHigh > 0:
```

Comment:
Yeah I did think that while coding it, but something made me do it this way. I'll change the logic there